third_party/jsoncpp/CMakeLists.txt

```diff
@@ -28,9 +28,10 @@ endforeach()
 
 ExternalProject_Add(
     jsoncpp
-    GIT_REPOSITORY https://github.com/open-source-parsers/jsoncpp
-    GIT_TAG 1.8.4
+    GIT_REPOSITORY https://github.com/sergeyrachev/jsoncpp
+    GIT_TAG cmake-config-improvements
+    #GIT_REPOSITORY https://github.com/open-source-parsers/jsoncpp
+    #GIT_TAG 1.8.4
```

Comment:
This is the part that will require a new release upstream (instead of using Sergey's PR)